charts/gardener-extension-admission-gcp/charts/runtime/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if not .Values.global.virtualGarden.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "name" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "labels" . | indent 4 }}
+{{- else if and .Values.global.virtualGarden.enabled .Values.global.virtualGarden.user.name }}
+{{- if .Values.global.serviceAccountTokenVolumeProjection.enabled }}
+apiVersion: v1
```

Comment:
Do we need a separator here above `---`?

Reply:
No. Both conditions cannot be fulfilled at the same time.